algo/src/test/java/com/hartwig/actin/algo/evaluation/general/HasMaximumBMITest.java

```diff
@@ -0,0 +1,67 @@
+package com.hartwig.actin.algo.evaluation.general;
+
+import com.hartwig.actin.algo.datamodel.Evaluation;
+import com.hartwig.actin.algo.datamodel.EvaluationResult;
+import com.hartwig.actin.clinical.datamodel.ImmutableBodyWeight;
+import org.junit.Test;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Collections;
+
+import static com.hartwig.actin.algo.evaluation.EvaluationAssert.assertEvaluation;
+import static org.junit.Assert.assertTrue;
+
+public class HasMaximumBMITest {
+    private final HasMaximumBMI function = new HasMaximumBMI(40);
+    private final LocalDate now = LocalDate.now();
+    private final LocalDate lastYear = now.minusYears(1);
```

Comment:
Do you prefer static or non-static here? I am fine either way, and happy to become consistent with your preference. Previously I have made all test constants static.

Reply:
On second thought, I think I prefer these to be non-static and re-initialized for every test execution. The test class doesn't need to be aware of the setup for `HasMaximumBMI`, so it's safest to create it anew for each test case. The time-based values could theoretically be different for each test (this would be more relevant if we were dealing with milliseconds instead of days), so I think it's appropriate to leave them non-static as well.

Reply:
Agreed